ui/app/components/database-connection.js

```diff
@@ -78,18 +79,18 @@ export default class DatabaseConnectionEdit extends Component {
   }
 
   @action
-  continueWithRotate() {
+  @waitFor
```

Comment:
oddly enough this was necessary to resolve failing  tests `database/workflow-test.js`. i didn't double check but my guess is that something within `HDS::Modal`s changed with this latest update.